### DISCUSSION TITLE
how to disable selinux when make build

### DISCUSSION THREAD

**Discussion by miaohao0309** [opencontainers/runc#4567] [Q&A]
i want disable selinux when make build，help me，thanks

**Comment by cyphar** [comment]
Currently there is no way to compile runc without SELinux support (as mentioned in the readme, this was removed in 1.0.0-rc93). Is there a particular reason you'd like to compile runc this way?

The SELinux support code should have no effect on non-SELinux systems. And since the SELinux code is written in pure Go, there is no real downside to including it (ditto for AppArmor).
